lib/paper_trail/config.rb

```diff
@@ -32,7 +32,9 @@ def serialized_attributes=(value)
     end
 
     def track_associations
-      @track_associations ||= PaperTrail::VersionAssociation.table_exists?
+      @track_associations.nil? ?
+        PaperTrail::VersionAssociation.table_exists? :
+        @track_associations
```

Comment:
> We could actually simplify this to @track_associations && PaperTrail::VersionAssociation.table_exists?, right?

That conjunction would require a database connection, and thus would not fix https://github.com/airblade/paper_trail/issues/636